rpms/SPECS/wazuh-agent.spec

```diff
@@ -334,7 +335,6 @@ SCA_BASE_DIR="%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp"
 mkdir -p %{_localstatedir}/ruleset/sca
 
 SCA_TMP_DIR="${SCA_BASE_DIR}/${SCA_DIR}"
-
```

Comment:
NIS: We shouldn't remove this blank line.

Reply:
Addressed https://github.com/wazuh/wazuh-packages/pull/2299/commits/017d620c5aed9a8231c49bed65e78dc9702adadd